install.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "$BIN_DIR" ]; then
+  BIN_DIR=$(pwd)
+fi
+
+THE_ARCH_BIN=""
+THIS_PROJECT_OWNER="dunglas"
+DEST=$BIN_DIR/frankenphp
```

Comment:
Why not install directly the binary in `/usr/local/bin/`? This will simplify the one-liner provided in docs.